boards/arm/efr32xg24_dk2601b/Kconfig.defconfig

```
@@ -40,6 +40,12 @@ choice BT_HCI_BUS_TYPE
 	default BT_SILABS_HCI
 endchoice
 
+if BT_SHELL
+	config SHELL_STACK_SIZE
+		default 4096
+
+endif # BT_SHELL
```

Not sure if this is the right solution or not, but looking at other boards and what they've done they have done it like
```suggestion
if SHELL

config SHELL_STACK_SIZE
	default 4096
	
endif # SHELL
```

i.e. using `SHELL` instead of `BT_SHELL`